terraform/README.md

```diff
@@ -0,0 +1,48 @@
+# Cloudflare Infrastructure
+
+This folder manage Node.js Cloudflare settings using Terraform
+
+### Contributing
+
+To modify the Cloudflare settings, you must fork/clone this repository and submit a pull request with the changes. Any alterations made in the `main` branch will be deployed to the Cloudflare account automatically. So, the pull request is required to review the changes before they are deployed.
+
+### Historical Context
+
+Today, we use Terraform to manage DNS records in Cloudflare. Previously, we used the Cloudflare UI for this task. To begin using Terraform, we cloned the Cloudflare settings and migrated them as the initial Terraform state using the utility cf-terraforming. This step was completed only once, and the state was stored in Terraform Cloud.
+
+Since the imported resources had non-human friendly names like "terraform_managed_resource_*," we cannot change their names to prevent recreation or updates of the resources. However, we can use our own naming conventions for new Terraform resources, and there is no need to run the cf-terraforming utility again.
+
+
+### How to review changes in a PR?
+
+The terraform plan is generated automatically in the PR. You can review the changes in the `terraform plan` section in the PR.
+
+There are two possible scenarios:
+- No changes
+- Changes required
+
+When changes are required you can see the details in the `terraform plan` section. Here you have several isolated examples of the changes:
+- [Add resource](https://github.com/UlisesGascon/poc-nodejs-cloudflare-terraform/pull/5)
```

Comment:
NIT, add a TODO to replace links to PRS in this repo?

Reply:
It is a great suggestion, right now `UlisesGascon/poc-nodejs-cloudflare-terraform` is archived, so the PRs still visible. We can replace them in the future when we have the first changes made from Terraform in this repository.

I created a subtask for it in https://github.com/nodejs/build/issues/3270#issuecomment-1600513990 👍